dev.sh

```diff
@@ -1,5 +1,46 @@
 #!/bin/bash
 
+REQUIRED_NODE_VERSION=22.9.0
+REQUIRED_YARN_VERSION=1.22.22
+
+version_ge() {
```

Comment:
Was this supposed to be `version_get`?
Or maybe you meant greater-than or equal-to.

IMO naming the function `compare_version` or `version_at_least` or something more obvious would be better.

Reply:
I have now changed it to `version_at_least`